README.md

```diff
@@ -173,6 +173,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Viveka Health](http://www.vivekahealth.com/) | Sanepa, Lalitpur | Software company working on health data projects
 - [Webpage Nepal Pvt. Ltd.](http://webpagenepal.com/) | New Road, Pokhara | Pokhara based web development company
 - [WebRobo](https://www.web-robo.com/) | Setopul, Kathmandu | Creative website development and digital marketing
+- [WebRoot Nepal](https://www.webrootnepal.com)| Tinkune, Butwal |  Website development, designing, hosting   
```

Comment:
Hello @bishalrijal, Thank you for the addition. Nice to see a tech company from Butwal. 🙂 

However, the website doesn't seem to have any real content. Please let me know when you have an updated website. The website is not helpful right now.